src/main/java/jp/co/soramitsu/sora/didresolver/commons/DIDTypeEnum.java

```diff
@@ -6,11 +6,11 @@
 
 @AllArgsConstructor
 public enum DIDTypeEnum {
-    IROHA("\\w*@\\w*\\..*"),
-    UUID("[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}"),
-    ED("\\w*");
+  IROHA("(?<ACCOUNT>\\w+)@(?<DOMAIN>\\w+[\\w|.]*)"),
+  UUID("[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}"),
+  ED("\\w+");
```

Comment:
>“ed” did-type prevents this by requiring to use a base58 encoded Ed25519 public key as a DID.

this means that for ED to be valid it is not enough to have just `\\w+` validity :) 

However, maybe it would be good to temporarily remove ED type. We will come back to it later.

Reply:
Do I remove it now?

Reply:
yes